guidelines/content-guidelines/01-content-strategy-k2.md

```diff
@@ -0,0 +1,193 @@
+> In its simplest form, troubleshooting information follows this pattern:
+>
+> 1. A condition or symptom. Usually the condition or symptom is an undesirable state in a system, a product, or a service that a reader wants to correct.
+> 2. A cause for the condition or symptom.
+> 3. A remedy for the condition or symptom.
+>
+> The troubleshooting topic provides sections for describing the condition, causes, and remedies needed to restore a system, a product, or a service to normal.
+>
+> For some conditions there could be more than one cause-remedy pair. The troubleshooting topic accommodates this. Typically, a cause is immediately followed by its remedy. Multiple cause-remedy pairs can provide a series of successive fall-backs for resolving a condition.
+>
+> Cause and remedy might occur in combinations other than pairs. It is possible to have:
+>
+> - Multiple causes with the same remedy
+> - A single cause with more than one remedy
+> - A remedy with no known cause
+> - A cause with no known remedy
+> The troubleshooting information type also can be used to document alarm clearing strategies.
+
+You can use the [troubleshooting topic template](../templates/resources/troubleshooting.md).
+
+### Release Notes
+
+Announce what's new in Kyma.
+
+After an introductory paragraph that outlines the city that's the namesake of the current release, a list briefly presents the new and changed features. Links lead to longer paragraphs that describe the changes in more detail.
+
+## Graphics
+
+### Architecture and Flow Diagrams
+
+For information about our diagram style, see [Diagrams](../02-diagrams.md).
+
+### Screenshots
+
+Wherever possible, present screenshots as simplified user interfaces (SUI). Basically, this means to blur out all UI elements that aren't essential for the task at hand.
+
+For more information, see:
+• <https://www.techsmith.com/blog/simplified-user-interface/>
+• <https://www.tcworld.info/e-magazine/technical-writing/simplified-graphics-and-screenshots-in-software-documentation-1102/>
+
+## Target Groups
+
+The general assumption is that the audience is familiar with the following terms and does not require the explanation of technical concepts behind them:
+
+- Kubernetes
+- Docker and containers
+
+<!-- Information about generic target groups is based on the SAP Styleguide: https://help.sap.com/viewer/DRAFT/e33c591ae4494a659a3f5f983c9d1161/PROD/en-US/546dfb06e80c4005aabc4795d548fe35.html -->
+
+### Decision maker
+
+**Tasks**: Assess the software to make sure that it will meet the company's needs. Needs the facts – not just marketing spin – before signing on the dotted line.
+**Goal**: Purchase the right solution for the company, and ensure stakeholders back this decision.
+
+### Software developer
+
+**Tasks**: Interested in technical topics. Solid knowledge of programming languages. Experienced in programming and development projects. Expert in the technical or business area. Uses and contributes to community content.
```

Comment:
Is "being interested" a task?
Maybe we should change `**Tasks:**` to `**Characteristics:**` or `**Who:**`?

Reply:
How about "stay up to date with technical topics" ?

Reply:
Well, actually, it was an example, as all of the "tasks" for the software developers are not really tasks, but characteristics. Maybe apart from `Uses and contributes to community content.`.
But we can turn a blind eye to this if it's just me that's bothered by this technicality.

Or, as you suggested, we could rephrase them to make them tasks :)

Reply:
... or we can totally get rid of both `Tasks` and `Goal` and just provide descriptions. If we choose to stay with the initial structure, I'd start each "sub-section" with a new line. They display in a single line in GitHub preview:

<img width="1053" alt="Screen Shot 2021-08-10 at 13 24 20" src="https://user-images.githubusercontent.com/35192450/128858775-40c596d5-c9d3-4d73-b2fb-700d13d4c537.png">

Reply:
good idea, done!